### ISSUE TITLE
colors in grapghs and color blindness

### ISSUE THREAD

**Issue by MarjoleinSchwachofer** [quintel/etmoses#1324]
I talked to someone with color blindness and he cannot see the difference between purple and blue in the graphs of ETMoses. An enhancement would be to change  the colors from the graphs

**Comment by ChaelKruip** [comment]
> An enhancement would be to change the colors from the graphs

Good idea. Do you have suggestions for the colours @MarjoleinSchwachofer?

**Comment by MarjoleinSchwachofer** [comment]
I have no idea myself, other than that the colors must be different in contrast. 
When my collegue is back from vacation I will ask him if he has ideas.

**Comment by MarjoleinSchwachofer** [comment]
Atleast **not** red with green, or blue withpurple. These are problem combinations

**Comment by ChaelKruip** [comment]
> When my collegue is back from vacation I will ask him if he has ideas.

👍 

[This website](https://designshack.net/articles/accessibility/tips-for-designing-for-colorblind-users/) explains some things to take into account.

**Comment by antw** [comment]
The "pop-over" legend will now show each series name. We should still revisit the colours, but hopefully this is a small step in the right direction:

<img width="185" alt="screen shot 2016-07-27 at 11 43 58" src="https://cloud.githubusercontent.com/assets/4383/17172904/7cd040b0-53ef-11e6-9a94-fc3e5b8a0601.png"> <img width="419" alt="screen shot 2016-07-27 at 11 35 40" src="https://cloud.githubusercontent.com/assets/4383/17172906/811c9286-53ef-11e6-945d-e0f5151a96f7.png">
